windows/hid.c

```diff
@@ -756,6 +756,28 @@ static struct hid_device_info *hid_internal_get_device_info(const wchar_t *path,
 	return dev;
 }
 
+static int hid_blacklist(unsigned short vendor_id, unsigned short product_id)
```

Comment:
We use also such a denylist in our app. But we use more filter criteria: USB-Interface-Number, Usage-Page, Usage and Bus-Type

Reply:
Yeah, that logic could be moved into SDL, but since we've definitely run into problems with those devices I thought it would be worth sharing upstream.